test/clj/rems/test_pdf.clj

```diff
@@ -40,6 +40,76 @@
                    (#'pdf/render-fields data)))))))
     (is (some? (with-language :en #(pdf/application-to-pdf-bytes data))))))
 
+;; reviewer should not be able to see applicant private form fields
+(deftest test-pdf-reviewer-private-fields
```

Comment:
this test feels a little heavy for such simple test, but i'm not sure how to incorporate the private field into existing `test-pdf-gold-standard`. it would require different user and the existing test is anyway meant for exhaustive rendering, while this case feels more like regression testing for a specific use case. heaviness, i guess, comes from all the upfront work needed.

Reply:
I personally think it is good the way it is.

Reply:
implemented more thorough pdf rendering test cases with public/private forms & fields with applicant/handler/reviewer/decider combination. this should serve as a basis for the second ticket privacy features (https://github.com/CSCfi/rems/issues/2042)